src/forms/ThresholdsSelection.css

```diff
@@ -1,8 +1,10 @@
-.Threshold {
-  width: 100%;
-  border: 1px solid #a5a5a5;
-  padding: 5px;
-  margin: 0 0 10px 0;
+th {
```

Comment:
Styling th and td like this has an effect on all th and td of the whole page.
Somehow the react css import doesnot scope those.
I struggle to find the documentation on this unfortuanedly..
But you can find the prove of it by inspecting that the css selectors in the browser are not prefixed.
Very confusing part of css modules...

A solution would be to use class selectors in combination with element selectors.
Check css combinators.
Or you could use just only classes.
Please let me know if you have questions about this.

Reply:
Ah you are right. This is so annoying indeed :(. I added a class selector for it.